<launch>
	<node pkg="spirecv_ros" type="rtsp2topic" name="rtsp2topic" output="screen">
	</node>

    <node pkg="spirecv_ros" type="mbzirc_suav_vessel_detection" name="mbzirc_suav_vessel_detection" output="screen">
        <param name="mode" type="string" value="all" />
        <param name="camera_parameters" type="string" value="$(find spirecv_ros)/camera_params/calib_webcam_640x480.yaml" />
        <param name="local_saving_path" type="string" value="$(find spirecv_ros)/video_vessel" />
        <param name="local_saving_path_2" type="string" value="$(find spirecv_ros)/video_usv" />
        <param name="show_image" type="bool" value="True" />
        <param name="save_video" type="bool" value="True" />
    </node>

</launch>
